.github/workflows/build_and_test.yml

```diff
@@ -167,6 +167,38 @@ jobs:
         cd Examples/Tutorials/Tutorial-HelloWorld
         make -f Makefile.non_windows all
         ./Tutorial-HelloWorld
+        
+  macos-apple-silicon-arm64:
```

Comment:
I've updated `build_and_test.yml` and it now triggers on PR. You can sync this branch to PcapPlusPlus `dev` and it should trigger for the PR

Reply:
Thank you! Done.